src/main/java/org/mybatis/spring/annotation/MapperScan.java

```diff
@@ -31,6 +31,11 @@
 /**
  * Use this annotation to register MyBatis mapper interfaces when using Java Config. It performs when same work as
  * {@link MapperScannerConfigurer} via {@link MapperScannerRegistrar}.
+ * 
+ * <p>Either {@link #basePackageClasses} or {@link #basePackages} (or its alias
```

Comment:
Please indicate to support since 2.0.4.

Reply:
Thank you for the feedback. I added it in line:37